### PR TITLE
contentKey dialog property

### PR DESCRIPTION
**Pull request by RDMStreet**

### CODE REVIEW

File: src/components/Dialog/SDialog.vue [hunk 1]
```diff
@@ -132,6 +132,12 @@ export default class SDialog extends Mixins(BorderRadiusMixin) {
    * `(done: boolean) => {}`
    */
   @Prop({ type: Function }) readonly beforeClose!: (done: boolean) => {}
+  /**
+   * Key for dialog content.
+   *
+   * `` by default
+   */
+  @Prop({ default: '', type: String || Number }) readonly contentKey!: string | number
```
Comment (stefashkaa) [inline src/components/Dialog/SDialog.vue:140]: `type: [String, Number]`

File: src/components/Dialog/SDialog.vue [hunk 2]
```diff
@@ -132,6 +132,12 @@ export default class SDialog extends Mixins(BorderRadiusMixin) {
    * `(done: boolean) => {}`
    */
   @Prop({ type: Function }) readonly beforeClose!: (done: boolean) => {}
+  /**
+   * Key for dialog content.
+   *
+   * `` by default
```
Comment (stefashkaa) [inline src/components/Dialog/SDialog.vue:138]: Remove this line please

File: src/components/Dialog/SDialog.vue [hunk 3]
```diff
@@ -132,6 +132,12 @@ export default class SDialog extends Mixins(BorderRadiusMixin) {
    * `(done: boolean) => {}`
    */
   @Prop({ type: Function }) readonly beforeClose!: (done: boolean) => {}
+  /**
+   * Key for dialog content.
```
Comment (stefashkaa) [inline src/components/Dialog/SDialog.vue:136]: Please add something like: 'If you need force rerender of table content (for instance, columns were changed), you should set some condition value here'

Comment (RDMStreet) [reply src/components/Dialog/SDialog.vue:138]: Done

File: src/components/Dialog/SDialog.vue (hunk 2 above)
Comment (RDMStreet) [reply src/components/Dialog/SDialog.vue]: Done

File: src/components/Dialog/SDialog.vue (hunk 1 above)
Comment (RDMStreet) [reply src/components/Dialog/SDialog.vue]: Fixed

**Review by stefashkaa — APPROVED** [review]

**Pull request merged**